sidebars.js

```diff
@@ -427,4 +427,32 @@ module.exports = {
       ],
     },
   ],
+  tctlReferenceMain: [
+    {
+      type: "category",
+      label: "tctl commands",
+      collapsible: false,
+      link: {
+        type: "doc",
+        id: "reference/tctl/index",
```

Comment:
This here is the trick.
As long as there is an "index.md" file in the directory, then Docusaurus will auto route "/docs/tctl/" to it - but without showing "index" in the url.
Then, if we add the index file to the sidebar as the category linked page - then we get the nice effect of having a fully clickable sidebar.

Reply:
Sweet...